locations/README.md

```diff
@@ -0,0 +1,33 @@
+## Add Locations
+Every Location is linked to a Brand, and the Brand is linked to the Program.
```

Comment:
kind of, but this is a bit confusing as you add locations to programs (on the dashbard) not brands, and you have to call `POST programs/{programId}/locations` to create a location. so i'd either rewrite or remove this sentence